src/main/webapp/resources/js/pages/projects/samples-metadata-import/components/SampleMetadataImportReview.tsx

```diff
@@ -204,28 +208,36 @@ export function SampleMetadataImportReview(): JSX.Element {
       .map((metadataItem) => metadataItem.rowKey);
 
     if (projectId) {
-      const response = await dispatch(
-        saveMetadata({ projectId, selectedMetadataKeys })
-      ).unwrap();
-
-      if (
-        Object.entries(response.metadataSaveDetails).filter(
-          ([, metadataSaveDetailsItem]) => metadataSaveDetailsItem.error
-        ).length === 0
-      ) {
-        navigate(`/${projectId}/sample-metadata/upload/complete`);
-      } else {
-        setLoading(false);
-      }
+      await dispatch(saveMetadata({ projectId, selectedMetadataKeys }))
+        .unwrap()
+        .then(({ metadataSaveDetails }) => {
+          if (
+            Object.entries(metadataSaveDetails).filter(
+              ([, metadataSaveDetailsItem]) => metadataSaveDetailsItem.error
+            ).length === 0
+          ) {
+            navigate(`/${projectId}/sample-metadata/upload/complete`);
+          } else {
+            setLoading(false);
```

Comment:
A notification should be shown at this point indicating that some of the samples failed.

Reply:
Please see 8f6bde80e99f4db32664cf7a413c76e4e7f206ed.